.github/workflows/refresh_db.yml

```diff
@@ -2,8 +2,8 @@ name: DB Refresh (Objects API)
 
 on:
   schedule:
-#    - cron: '*/20 * * * *'  # every 20 minutes
     - cron: '0 5 * * *'  # every day 5am GMT (12-1am EST/EDT)
+#    - cron: '*/20 * * * *'  # every 20 minutes
```

Comment:
### "Fetch failures - 0 members": Now 1x/day
It is not strictly necessary that it runs 1x/day but it is a good failsafe, especially for now.

Reply:
### "Fetch failures - 0 members": It looked like was broken, but don't think it was
**Why I thought it was broken:**
I was surprised to see that there was 1 unresolved case in `fetch_audit`: `121711287 (Hope Termhub Test (v5))`

**Why I think it's not broken:**
I checked DB refresh nightly logs for the last 7 days (it actually was not broken; just some days had no new csets), this is the only one that did not appear.

This indicates that hope fetched this using the refresh button on dev or prod.

And I'm thinking that when she clicked that button, that was the version of the codebase where it did add it to the `fetch_audit` log, but before my "fetch failures - 0 members" was implemented. So it didn't get resolved.

---

Given this, it does not indicate that it is strictly necessary to have "fetch failures - 0 members" run nightly, but I think (and I think you agree) that we should activate it nightly as a failsafe, especially for now, so I've done that.